src/chttp.cpp

```diff
@@ -214,8 +229,9 @@ LUA_FUNCTION(CHTTP) {
 		request->method = METHOD_GET;
 	}
 	if (request->method == METHOD_NOSUPP) {
-		runFailedHandler(LUA, request->failed, "Unsupported request method: " + std::string(LUA->GetString(-1)));
+		runFailedHandler(LUA, request->success, request->failed, "Unsupported request method: " + std::string(LUA->GetString(-1)));
```

Comment:
`success` is set last, and I'm not sure what the behavior for not-set variables is. Maybe we should just hardcode it to zero in this case (and in the similiar call below)?